src/formatDate.js

```diff
@@ -51,6 +51,63 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const NEW_DIVIDER = toFormat[3];
```

Comment:
I think it is better to name the variable like 'separator' for better understanding)